css-layout-classes/styles.css

```diff
@@ -0,0 +1,33 @@
+/* element selectors */
+* {
+  font-family: sans-serif;
+}
+
+img {
+  width: 100%;
+}
+
+/* class selectors */
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.two-thirds {
+  flex-basis: calc(100% - (100%/3));
+}
+
+.one-fourth {
+  flex-basis: 25%;
+}
```

Comment:
You should rename these classes to include `column` so that it is clear they are layout classes

Reply:
changed the names to reflect that they are columns.